deploy/examples/create-external-cluster-resources.py

```diff
@@ -266,17 +266,17 @@ def gen_arg_parser(cls, args_to_parse=None):
         common_group.add_argument("--run-as-user", "-u", default="", type=str,
                                   help="Provides a user name to check the cluster's health status, must be prefixed by 'client.'")
         common_group.add_argument("--cluster-name", default="",
-                                  help="Ceph cluster name")
+                                  help="Ceph cluster name(should be small letter alphabets)")
```

Comment:
Can we always convert to lower case instead of asking for it?

Reply:
Done